### PR TITLE
added language support

### LINKED ISSUE #1002: Multilingual support : Adding language support to generated Webapp
After generating app, participant can change the language of the pages with a selectbox.
Things to Consider
1. The webapp generated are static pages that work offline.

@Princu7 @mariobehling What are your views ?

### PR DESCRIPTION
**Pull request by utkarshnath**
fixed #1002 , deployed on Heroku
https://desolate-meadow-51072.herokuapp.com/

@aayusharora plz review

### CODE REVIEW

File: src/backend/templates/partials/navbar.hbs [hunk 1]
```diff
@@ -43,7 +43,23 @@
                 <li class="pull-left social-icons"><a href="{{link}}" style="padding-right:0; padding-left:0; margin-left:15px"><i class="fa fa-lg fa-{{icon}}" aria-hidden="true" title="{{{icon}}}"></i></a></li>
               {{/if}}
             {{/sociallinks}}
-
+              <li class="google_translator">
+                  <div id="google_translate_element"></div>
+                  <script type="text/javascript">
+                      function googleTranslateElementInit() {
```
Comment (aayusharora) [inline src/backend/templates/partials/navbar.hbs]: Can we make a separate js file to handle this @utkarshnath ?

File: src/backend/_scss/application.scss [hunk 2]
```diff
@@ -93,6 +93,24 @@ a {
...
+  .goog-te-gadget {
+    color: transparent !important;
+  }
+}
+
+.goog-te-banner-frame.skiptranslate {
+  display: none !important;
+}
+body {
+  top: 0px !important;
```
Comment (aayusharora) [inline src/backend/_scss/application.scss]: @utkarshnath  please remove `px`

**Comment by aayusharora** [comment]
@utkarshnath One suggestion is to use bootstrap in-built select box.
And also make sure that the design does not break on mobile. 
Thanks.

**Comment by utkarshnath** [comment]
@aayusharora we can't really integrate bootstrap in-built select box with google translate widget

**Comment by aayusharora** [comment]
@utkarshnath we can integrate a class in it even if the divs are appending dynamically, by making sure that our function will run after google translate. 
Well to make it more simpler, we can write custom CSS on automatically added google translate classes or tags.

**Comment by utkarshnath** [comment]
@aayusharora thank you for your suggestions. I was making some kind of error initially. I have integrated bootstrap with Google Translate widget and taken care that design doesn't break on mobile.
pls check at https://guarded-inlet-13297.herokuapp.com/

**Comment by utkarshnath** [comment]
@aayusharora please review, I have made a separate js file for translation.

**Comment by aayusharora** [comment]
@utkarshnath Thanks !! But the link https://desolate-meadow-51072.herokuapp.com/ is not working for me, hanging at 70% if API end point is given.

**Comment by utkarshnath** [comment]
@aayusharora Pls check this https://guarded-inlet-13297.herokuapp.com/ 
I have tested it with https://eventyay.com/api/v1/events/6

**Comment by aayusharora** [comment]
Hey @utkarshnath Cool !! Looks good.
- Just make sure you have pulled the new code as I have seen some old things in this PR.
- Can you please check https://www.codacy.com/app/dev_19/open-event-webapp/pullRequest?prid=533211. You can improve some part of it to pass the codacy check.
Thanks

**Pull request closed without merging**